src/jit/emit.cpp

```diff
@@ -1082,6 +1083,102 @@ int emitter::instrDesc::idAddrUnion::iiaGetJitDataOffset() const
     return Compiler::eeGetJitDataOffs(iiaFieldHnd);
 }
 
+#if defined(DEBUG) || defined(LATE_DISASM)
+
+#if 0  // ndef _TARGET_XARCH_
```

Comment:
It seems that you have a minimal implementations of these for arm, and actual implementations for xarch and arm64, so perhaps this should just be removed?

Reply:
OK will do.  Most of the Arm64 implementation is still being worked on.